src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="stopwatch__seconds"></div>
+      <div class="stopwatch__minutes"></div>
+      <div class="stopwatch__circle"></div>
```

Comment:
According to the checklist, BEM methodology should be used for class names. The class `stopwatch--speed-up` seems to be a modifier, but there is no base class `stopwatch` on the same element. You should add a base class `stopwatch` to the div element if `stopwatch--speed-up` is a modifier.